### PR TITLE
added a new changelog post for spin cloud v0.4.0

### PR DESCRIPTION
**Pull request by macolso**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title, `template`, and `date` are all set
- [x] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)

Screenshot 2023-10-13 at 11 20 51

- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)

Screenshot 2023-10-13 at 11 21 44

- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors

Screenshot 2023-10-13 at 11 23 24

- [x] Relates to an existing (potentially outdated) blog article? If so please add URL in blog to point to this content.

### CODE REVIEW

**Review by kate-goldenring — COMMENTED** [review]
Do we also want to mention we are now using OCI instead of bindle?

**Comment by itowlson** [comment]
Wasn't OCI part of 0.3?  If it's not in the 0.3 changelog we should retcon something in, because of the "once you're deployed or updated an app with 0.3 you can no longer update it with 0.2" thing

**Comment by mikkelhegn** [comment]
I don't think we ever did a 0.3.0 change log. So either add that here as well for full reference, or make this a 0.4.0 and 0.3.0 combined change log post.

File: content/cloud/changelog/spin-cloud-v0.4.0.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+title = "Fermyon Serverless AI (Private Beta)"
```
Comment (flynnduism) [inline content/cloud/changelog/spin-cloud-v0.4.0.md:1]: THis title seems to be cloned from the prior changelog entry, should be updated to be about spin cloud plugin

**Comment by macolso** [comment]
> I don't think we ever did a 0.3.0 change log. So either add that here as well for full reference, or make this a 0.4.0 and 0.3.0 combined change log post.

This is our first ever change log post for a `spin cloud` version update. I can expand it to include 0.3.0 - 0.4.0.

**Comment by taylols** [comment]
spin cloud graphic added ✔️

spin-cloud-0 4 0

File: content/cloud/changelog/spin-cloud-v0.4.0.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Some highlights include:
+
+* `spin cloud link sqlite` will connect your NoOps SQL DB to your Spin application of choice
+* `spin cloud unlink sqlite` will disconnect your NoOps SQL DB to your Spin application of choice
+* `spin cloud sqlite create` will create a NoOps SQL database on Fermyon Cloud
+* `spin cloud sqlite rename` will rename your NoOps SQL database
+
+> Note that NoOps SQL Database is currently in private beta. You can request access using this [short form](https://fibsu0jcu2g.typeform.com/to/Brv12FI0#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx).
+
+<img src="TODO - add image" alt="spin cloud plugin v0.4.0">
```
Comment (tpmccallum) [inline content/cloud/changelog/spin-cloud-v0.4.0.md:32]:
```suggestion
```

File: content/cloud/changelog/spin-cloud-v0.4.0.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+enable_shortcodes = true
+tags = ["spin_cloud_plugin"]
+[extra]
+type= "changelog_post"
+twitter_card_type = "summary_large_image" 
+image = "/static/image/changelog/spin-cloud-0.4.0.jpg" 
+
+---
+
+The past month, we've released two minor versions of the [`spin cloud` plugin](https://github.com/fermyon/cloud-plugin). Let's take a look at what's new!
```
Comment (tpmccallum) [inline content/cloud/changelog/spin-cloud-v0.4.0.md:13]:
```suggestion
In the past month, we've released two minor versions of the [`spin cloud` plugin](https://github.com/fermyon/cloud-plugin). Let's take a look at what's new!
```

File: content/cloud/changelog/spin-cloud-v0.4.0.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+### As of spin cloud v0.3.0
+
+As of [spin cloud v0.3.0](https://github.com/fermyon/cloud-plugin/releases/tag/v0.3.0), Fermyon Cloud has removed its dependency on bindle and is now using OCI as the default registry. No action is needed on your part to modify your Spin applications. If you run into any issues, please reach out to us on [Discord](https://discord.gg/AAFNfS7NGf) for assistance.
+
+### As of spin cloud v0.4.0
+
+With [`spin cloud` plugin v0.4.0](https://github.com/fermyon/cloud-plugin), we've new functionality to help you manage your Spin application and NoOps SQL Database amoung other exciting changes.
```
Comment (tpmccallum) [inline content/cloud/changelog/spin-cloud-v0.4.0.md:21]:
```suggestion
With [`spin cloud` plugin v0.4.0](https://github.com/fermyon/cloud-plugin), we've new functionality to help you manage your Spin application and NoOps SQL Database, among other exciting changes. 
```

**Review by tpmccallum — APPROVED** [review]

File: content/cloud/changelog/spin-cloud-v0.4.0.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+Some highlights include:
+
+* `spin cloud link sqlite` will connect your NoOps SQL DB to your Spin application of choice
+* `spin cloud unlink sqlite` will disconnect your NoOps SQL DB to your Spin application of choice
+* `spin cloud sqlite create` will create a NoOps SQL database on Fermyon Cloud
+* `spin cloud sqlite rename` will rename your NoOps SQL database
+
+> Note that NoOps SQL Database is currently in private beta. You can request access using this [short form](https://fibsu0jcu2g.typeform.com/to/Brv12FI0#hubspot_utk=xxxxx&hubspot_page_name=xxxxx&hubspot_page_url=xxxxx).
+
+
```
Comment (tpmccallum) [inline content/cloud/changelog/spin-cloud-v0.4.0.md:32]:
```suggestion
<img src="/static/image/changelog/spin-cloud-0.4.0.jpg" alt="Spin Cloud 0.4.0 changelog">

```

File: content/cloud/changelog/spin-cloud-v0.4.0.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
+title = "Spin Cloud v0.3 & 0.4 Updates"
+template = "changelog_item"
+date = "2023-10-13T09:00:00Z"
```
Comment (tpmccallum) [inline content/cloud/changelog/spin-cloud-v0.4.0.md:3]:
```suggestion
date = "2023-10-12T12:00:00Z"
```

**Review by tpmccallum — APPROVED** [review]

**Pull request merged**